examples/tutorials/self-paced-training/part-3_security_and_privacy/chapter-6_Security_in_federated_compute_system/06.2_identity_security/identity_security.ipynb

```diff
@@ -0,0 +1,210 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# **Indentity Security**  \n",
+    ">note\n",
+    " This is section is the same as \n",
+    "[FLARE's Security documentation](https://nvflare.readthedocs.io/en/2.4/user_guide/security/identity_security.html)\n",
+    "\n",
+    "\n",
+    "## Authentication\n",
+    "\n",
+    "NVFLARE’s authentication model is based on Public Key Infrastructure (PKI) technology\n",
+    "\n",
+    "For the FL project, the Project Admin uses the Provisioning Tool to create a Root CA with a self-signed root certificate. This Root CA will be used to issue all other certs needed by communicating parties.\n",
+    "\n",
+    "Identities involved in the project (Server(s), Clients, the Overseer, Users) are provisioned with the Provisioning Tool. Each identity is defined with a unique common name. For each identity, the Provisioning Tool generates a separate password-protected Startup Kit, which includes security credentials for mutual TLS authentication:\n",
+    "\n",
+    "* The certificate of the Root CA\n",
+    "* The cert of the identity\n",
+    "* The private key of the identity\n",
+    "\n",
+    "Startup Kits are distributed to the intended identities:\n",
+    "* The FL Server’s kit is sent to the Project Admin\n",
+    "* The kit for each FL Client is sent to the Org Admin responsible for the site\n",
+    "* FLARE Console (previously called Admin Client) kits are sent to the user(s)\n",
+    "\n",
+    "To ensure the integrity of the Startup Kit, each file in the kit is signed by the Root CA.\n",
+    "\n",
+    "Each Startup Kit also contains a “start.sh” file, which can be used to properly start the NVFLARE application.\n",
+    "\n",
+    "Once started, the Client tries to establish a mutually-authenticated TLS connection with the Server, using the PKI credentials in its Startup Kits. This is possible only if the client and the server both have the correct Startup Kits.\n",
```

Comment:
suggest: "Once started, the FL client attempts to establish a mutual TLS or standard TLS connection with the FL server, using the PKI credentials in its Startup Kit. This is possible only if both the FL client and the FL server have the correct Startup Kits."